x-pack/plugins/observability_solution/apm/public/components/routing/app_root/apm_header_action_menu/anomaly_detection_setup_link.tsx

```diff
@@ -51,7 +50,7 @@ export function AnomalyDetectionSetupLink() {
     tooltipText = getNoJobsMessage(anomalyDetectionSetupState, environment);
     icon = 'machineLearningApp';
   } else if (anomalyDetectionSetupState === AnomalyDetectionSetupState.UpgradeableJobs) {
-    color = 'success';
+    color = 'accentSecondary';
```

Comment:
It feels a bit odd to swap the `success` color for `accentSecondary`. 
I assume there’s a particular reason behind this change—could you please clarify? 
Thanks!

Reply:
Yes, there's a reason for this change, you can take a look here https://github.com/elastic/kibana/issues/199715?reload=1?reload=1